Detectors/TPC/qc/src/Tracks.cxx

```diff
@@ -103,44 +111,53 @@ bool Tracks::processTrack(const o2::tpc::TrackTPC& track)
   // ===| histogram filling including cuts |===
   if (absEta < 1. && nCls > 60 && dEdxTot > 20) {
 
+    if (pt > 0) {
```

Comment:
Why don't you use directly `track.getQ2Pt()` ?

Reply:
Good point, I will change that.